Search.java

```diff
@@ -109,6 +109,7 @@ public static void main(String[] args) throws java.io.IOException{
 		bestOfGenChromo = new Chromo();
 		bestOfRunChromo = new Chromo();
 		bestOverAllChromo = new Chromo();
+        HashMap<Integer, Integer> phylo = new HashMap<Integer, Integer>();

```

Comment:
Should we change change this to HashMap<Integer, Array<Integer>, HashMap<Integer, Array<Integer>> where the int is the child's ID, the int array is the child's phenotype, the hashmap stores the parents' IDs and their genomes? We can also add other ints for start generation and end generation but those haven't been implemented yet in initialization.